client/settings/fraud-protection/advanced-settings/rule-toggle.tsx

```diff
@@ -11,12 +11,14 @@ import { ToggleControl, RadioControl } from '@wordpress/components';
 import './../style.scss';
 import FraudPreventionSettingsContext from './context';
 
+const isReviewEnabled = false;
```

Comment:
We should change this to use the feature flag.

Reply:
Updated on [98e56e0](https://github.com/Automattic/woocommerce-payments/pull/6275/commits/98e56e0d36d5ac17d9937b95f57ce8c0a74c269b)!